### PR TITLE
Compare arrays

### PR DESCRIPTION
**Pull request by greggman**
There doesn't appear to be a test of comparing arrays

### CODE REVIEW

**Comment by greggman** [comment]
hold on. extra files should not be there

File: sdk/tests/conformance2/glsl3/array-equality.html [hunk 1]
```diff
@@ -77,7 +93,37 @@
...
+{
+  fShaderId: "simple-array-fs",
+  fShaderSuccess: true,
+  linkSuccess: true,
+  render: true,
+  uniforms: [
+    { name: 'a', functionName: 'uniform3fv', value: [1, 2, 3, 4, 5, 6, 7, 8, 9]},
+    { name: 'b', functionName: 'uniform3fv', value: [1, 2, 3, 4, 5, 6, 7, 8, 9]},
+  ],
+  passMsg: "Simple array with set values",
```
Comment (kenrussell) [inline sdk/tests/conformance2/glsl3/array-equality.html:125]: set -> same

File: sdk/tests/conformance2/glsl3/array-equality.html [hunk 2]
```diff
@@ -61,6 +61,22 @@
...
+uniform vec3 a[3];
+uniform vec3 b[3];
+
+out vec4 fragColor;
+
+void main(void) {
+    fragColor = vec4(0.0, 0.0, 0.0, 1.0);
+
+    if (a == b) {
+        fragColor.y = 1.0;
```
[the remark this answers is not in the document]
Comment (greggman) [reply sdk/tests/conformance2/glsl3/array-equality.html]: done

File: sdk/tests/conformance2/glsl3/array-equality.html (hunk 1 above)
Comment (greggman) [reply sdk/tests/conformance2/glsl3/array-equality.html]: done

**Comment by greggman** [comment]
I don't have commit access

**Review by kenrussell — APPROVED** [review]
Thanks for making the updates. I'll add you to the WebGL Team so you can merge things yourself.

**Pull request merged**